packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -4754,3 +4757,48 @@ export class Gitxnas extends Gtxnas {
 		super.execute(stack);
 	}
 }
+
+/**
+ * Takes the last value from stack and if base64encoded, decodes it acording to the
+ * encoding e and pushes it back to the stack, otherwise throws an error
+ */
+export class Base64Decode extends Op {
+	readonly line: number;
+	readonly encoding: Base64Encoding;
+
+	/**
+	 * Asserts 1 argument is passed.
+	 * @param args Expected arguments: [e], where e = {0, 1}.
+	 * @param line line number in TEAL file
+	 */
+	constructor(args: string[], line: number) {
+		super();
+		this.line = line;
+		assertLen(args.length, 1, line);
+		this.encoding = Number(args[0]);
```

Comment:
Ahh, My bad. The opcode arguments should be {URLEncoding, StdEncoding}.

https://github.com/algorand/go-algorand/blob/5af8ec8506ec770bc3462b2f4f78a284ae22bf94/data/transactions/logic/eval_test.go#L3489